Build/Scripts/Package/Package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Any": {
+        "includeInNuget": true
+    },
+    "Library Assert": {
+        "includeInNuget": true
+    },
+    "Library Variable Storage": {
+        "includeInNuget": true
+    },
+    "Performance Toolkit": {
+        "includeInNuget": false,
+        "_comment": "Not enabled yet"
+    },
+    "Permissions Mock": {
+        "includeInNuget": true
+    },
+    "System Application": {
```

Comment:
What about _System Application Tests_ and _System Application Test Libraries_? Don't we need them in the package as well?